src/misc/lv_text.c

```diff
@@ -136,6 +136,37 @@ void lv_text_get_size(lv_point_t * size_res, const char * text, const lv_font_t
         size_res->y -= line_space;
 }
 
+bool lv_text_is_cmd(lv_text_cmd_state_t * state, uint32_t c)
```

Comment:
the first params of a widget's public function need to be the obj it self.

I think you can get the state from lv_label_t casted from lv_obj_t.

Reply:
Hi, thanks for the feedback, I checked the other `lv_text` interfaces (in `lv_text.h`) and none of them has the `lv_obj_t` as first parameter, maybe this interface is an exception? What do you think @kisvegabor

Reply:
lv_text is not for widgets, so we can't pass obj parameters.